stress-generator/values.yaml

```diff
@@ -0,0 +1,19 @@
+image:
+  repository: kupolua/stress-generator
+  tag: latest
+  env:
+    hostname: HOST
+    hostvalue: stress-demo-api.default.svc.cluster.local
```

Comment:
use short domain, stress-demo-api
and split deployment variable and api service variable,
so it will should be easier to update
it's hardcoded to specific namespace, so won't work with other namespaces
stress-demo-api